packages/platform-sdk-profiles/src/drivers/memory/wallets/wallet-transaction-service.test.ts

```diff
@@ -1171,24 +1213,33 @@ it("should broadcast multisignature transaction", async () => {
 
 	const id = await subject.signMultiSignature({
 		nonce: "1",
-		from: "DEMvpU4Qq6KvSzF3sRNjGCkm6Kj7cFfVaz",
+		signatory: new Signatories.Signatory(
+			new Signatories.MultiSignatureSignatory({
+				publicKeys: [
+					"02edf966159de0013ca5b99371c5436e78f22df0d565eceee09feb977fe49cb910",
+					"034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
+				],
+				min: 2,
+			}),
+		),
 		data: {
 			publicKeys: [
 				"02edf966159de0013ca5b99371c5436e78f22df0d565eceee09feb977fe49cb910",
 				"034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
 			],
 			min: 2,
 		},
-		sign: {
-			senderPublicKey: "034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-			multiSignature: {
-				publicKeys: [
-					"02edf966159de0013ca5b99371c5436e78f22df0d565eceee09feb977fe49cb910",
-					"034151a3ec46b5670a682b0a63394f863587d1bc97483b1b6c70eb58e7f0aed192",
-				],
-				min: 2,
-			},
-		},
+		// @TODO: we need to allow setting of a senderPublicKey when using MultiSignatureSignatory
```

Comment:
Need to check why it's setting the sender public key here, think because of ARK needing it for the initial transaction.